docs/features/blocks/contentblocks/card.rst

```diff
@@ -0,0 +1,28 @@
+Card Block
+==========
+
+Allows the user to fill out information to be displayed in a card. Crops the image to a 3:2 ratio. 
+
+.. note::
+    There is also a Card Grid Layout Block which groups cards into a grid and is a LayOut Block; 
+    this Card Block only displays a single card.
```

Comment:
"Card Grid Layout Block" in the note here could link to the actual doc page.

Also, "LayOut" looks a bit odd, I think "Layout" is the spelling elsewhere in the docs.

Reply:
Done. I used the https://www.sphinx-doc.org/en/master/usage/restructuredtext/roles.html#ref-role notation to do it.